CHANGELOG.md

```diff
@@ -1,5 +1,160 @@
 # Changelog
 
+## v6.1.0-preview.1 - 2018-03-23
+
+### Breaking Changes
+
+- Throw terminating error in `New-TemporaryFile` and make it not rely on the presence of the `TEMP` environment variable (#6182) (Thanks @bergmeister!)
+- Remove the unnecessary `AddTypeCommandBase` class from `Add-Type` (#5407) (Thanks @iSazonov!)
+- Remove unsupported members from the enum `Language` in `Add-Type` (#5829) (Thanks @iSazonov!)
+- Fix range operator to work better with character ranges (#5732) (Thanks @iSazonov!)
```

Comment:
The template for the announcements repo said that we would have an announcement for breaking changes.

Reply:
We should keep `Breaking Changes` here, but should also separately post to `Announcements` repo